README.md

```diff
@@ -1 +1,55 @@
-# irods_client_http_python
+# iRODS HTTP API Python Wrapper
+
+This is a Python wrapper for the [iRODS HTTP API](https://github.com/irods/irods_client_http_api). 
+
+Documentation for the endpoint operations can be found [here](https://github.com/irods/irods_client_http_api/blob/main/API.md).
+
+## Setup
+**NOTICE:** This project is not yet available through pip. To use, clone the repository into the desired location.
+```
+git clone https://github.com/irods/irods_client_http_python.git
+```
```

Comment:
Can this be installed as a pip package at all? Or are you saying that this is not available on, e.g. PyPI?

If it can be installed via pip, once cloned, you can install the local directory:
```bash
git clone https://github.com/irods/irods_client_http_python.git
python3 -m pip install ./irods_client_http_python
```

Or, alternatively, use a Git URL to install via pip, like this:
```bash
python3 -m pip install git+https://github.com/irods/irods_client_http_python
```

But those suggestions are moot if this is not a package (yet) :)

Reply:
Still hasn't been made into a pip package yet. Opened issue #36